MAPL_cfio/CMakeLists.txt

```diff
@@ -29,15 +29,13 @@ set (lib MAPL_cfio_${precision})
 
 esma_add_library (${lib}
   SRCS ${srcs}
-  DEPENDENCIES ${ESMF_LIBRARIES}
+  DEPENDENCIES esmf NetCDF::NetCDF_Fortran
   )
-target_include_directories (${lib} PUBLIC ${INC_NETCDF} ${INC_ESMF})
+add_dependencies (${lib} esmf NetCDF::NetCDF_Fortran)
```

Comment:
Excellent!

Reply:
My comment was regarding the use of the `NetCDF::NetCDF_Fortran`,  but this call should be `target_link_libraries(...)` not `add_dependencies()`

Reply:
Thanks for catching this, @tclune! The command
```
target_link_libraries (${lib} esmf NetCDF::NetCDF_Fortran)
```
is already added by the `ecbuild_add_library` macro, called by `esma_add_library()`. Re-adding it here would generate an error (_"The keyword signature for target_link_libraries has already been used ..."_).
I would suggest removing `add_dependencies()`.

Reply:
Change implemented in 12d86b3.